core/components/footer/index.tsx

```diff
@@ -2,7 +2,6 @@ import { removeEdgesAndNodes } from '@bigcommerce/catalyst-client';
 import {
   SiFacebook,
   SiInstagram,
-  SiLinkedin,
```

Comment:
According to [LinkedIns brand guidelines](https://brand.linkedin.com/policies), we can't use it's logo without explicit permission. The underlying library is complying with Microsoft's legal team request to remove from their library, so it's safe to assume we should just remove it for now.